prowler/providers/aws/services/iam/iam_service.py

```diff
@@ -685,6 +691,40 @@ def _list_entities_role_for_policy(self, policy_arn):
         finally:
             return roles
 
+    def _list_entities_for_policy(self, policy_arn):
+        logger.info("IAM - List Entities Role For Policy...")
+        try:
+            entities = []
```

Comment:
Please, make `entities` a dictionary with keys as User, Group, Role with a list for each entity.

Reply:
So we can show in the status extended the type of entities too.

Reply:
Done -> [1ee24fc](https://github.com/prowler-cloud/prowler/pull/5437/commits/1ee24fc407ed10fc7a10f812a073b597ae811992)